tests/foreman/api/test_repository.py

```diff
@@ -165,9 +165,6 @@ def test_positive_assign_http_proxy_to_repository(self):
         repo_b.sync({'async': True})
         result = ssh.command('grep -F {} /var/log/messages'.format(proxy_fqdn))
         assert result.return_code == 0
-        # Delete repositories
-        entities.Repository(id=repo_a.id).delete()
-        entities.Repository(id=repo_b.id).delete()
```

Comment:
Could you explain why this change is made?

Reply:
Because repositories are being synced in background and I cann't delete them at that time, and because of this test is failing.

Reply:
I wanted to say that perhaps we should update test to wait for repositories to sync, assert on that and then delete, but looking at this particular test purpose, I think you made right call. Syncing is checked elsewhere, we can ignore it here.